src/App.tsx

```diff
@@ -1,24 +1,150 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoError } from './components/TodoError/TodoError';
+import { Errors } from './types/Errors';
+import { Filter } from './types/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10236;
+
+const preparedTodos = (todosList: Todo[], selectedFilter: Filter): Todo[] => {
+  let filteredTodos = [...todosList];
+
+  switch (selectedFilter) {
+    case 'Active':
```

Comment:
It would be cool to put the names of states in a separate enum:
```ts
enum TodoState {
  Active: 'Active',
  Completed: 'Completed'
}
```
This will help us avoid using "magic values"